migrations/248_request_course_add_info_fields.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE course_requests
```

Comment:
The name of this file should be `248_course_requests__extra_columns__add.sql` or something like that. After the number the filename should include the exact table name.